mteb/benchmarks/benchmarks/benchmarks.py

```diff
@@ -2295,3 +2295,26 @@
   year = {2025},
 }""",
 )
+
+REAL_MM_RAG = Benchmark(
+    name="REAL_MM_RAG",
+    display_name="IBM Visual Document Retrieval",
+    tasks=get_tasks(
+        tasks=[
+            "RealMMRagFinReportRetrieval",
+            "RealMMRagFinSlidesRetrieval",
+            "RealMMRagTechReportRetrieval",
+            "RealMMRagTechSlidesRetrieval",
+        ],
+    ),
+    description="Realistic and multi-modal document retrieval benchmark.",
```

Comment:
This description is too short. Why should I prefer this over another VDR benchmark